core/src/main/java/org/apache/iceberg/IncrementalFileCleanup.java

```diff
@@ -264,6 +264,14 @@ public void cleanFiles(TableMetadata beforeExpiration, TableMetadata afterExpira
     LOG.warn("Manifests Lists to delete: {}", Joiner.on(", ").join(manifestListsToDelete));
     deleteFiles(manifestsToDelete, "manifest");
     deleteFiles(manifestListsToDelete, "manifest list");
+
+    if (!beforeExpiration.statisticsFiles().isEmpty()) {
+      Set<String> expiredStatisticsFilesLocations =
+          expiredStatisticsFilesLocations(beforeExpiration, afterExpiration);
+      LOG.warn(
+          "Statistics files to delete: {}", Joiner.on(", ").join(expiredStatisticsFilesLocations));
```

Comment:
I know it was warn level for the other cases in incremental file cleanup, but it seems like something we shouldn't keep following IMO because it's not really a warning case. We know we want to delete these files and it's intentional. 

I think this could be debug level or we can simply remove it.

Reply:
I made it to debug and other places too I have changed to debug log.